openembedded-core-mbl/scripts/contrib/create-update-payload.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
```

Comment:
I think this script should be located in `meta-mbl-distro/scripts` since it is MBL distro specific and I don't think we have plans to submit it to oe-core.